pkg/geo/geomfn/voronoi.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geomfn
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/geo"
+	"github.com/cockroachdb/cockroach/pkg/geo/geopb"
+	"github.com/cockroachdb/cockroach/pkg/geo/geos"
+)
+
+// VoronoiDiagram Computes the Voronoi Diagram from the vertices of the supplied geometry.
+func VoronoiDiagram(
+	a, env geo.Geometry, tolerance float64, onlyEdges bool,
+) (geo.Geometry, error) {
+	var envEWKB geopb.EWKB
+	if !env.Empty() {
```

Comment:
we should use a pointer for `env` -- an empty geometry may produce different results (e.g. `POLYGON EMPTY`) compared to a 0 value geo.Geometry.
can you also rename all the `a` to `g`?

Reply:
right, I guess nil check would be enough?